src/_locales/tr/messages.json

```diff
@@ -0,0 +1,442 @@
+{
+    "social_tooltip_pb_has_replaced": {
+        "message": "Privacy Badger bu ",
+        "description": "Beginning of tooltip shown over a replaced social button, e.g [Privacy Badger has replaced this ]Facebook[ button]."
+    },
+    "badger_status_block": {
+        "message": "Engellendi ",
+        "description": ""
+    },
+    "invalid_json": {
+        "message": "Geçersiz JSON dosyası.",
+        "description": ""
+    },
+    "options_remove_origin_confirm": {
+        "message": "Bu kaynağı Privacy Badger'dan kaldırmak istediğinize emin misiniz?",
+        "description": ""
+    },
+    "badger_status_noaction": {
+        "message": "Takip yok ",
+        "description": ""
+    },
+    "non_tracker_tip": {
+        "message": "Şu an Privacy Badger sadece 3. partilerin çerez, yerel HTML5 depolaması, veya tuval parmak izi kullanarak taramanızı takip edip etmediğini kontrol ediyor. Bazı adresler Privacy Badger tarafından algılanmayan yöntemler kullanıyor olabilir.",
+        "description": ""
+    },
+    "intro_hungry": {
+        "message": "Hiçbir şey Privacy Badger'ı acıktığında çerez yemekten alıkoyamaz!",
+        "description": ""
+    },
+    "options_dnt_policy_setting": {
+        "message": "Sitelerin EFF'nin Takip Etme poliçesi ile uyumluluğunu kontrol et",
```

Comment:
Should we leave [Do Not Track](https://en.wikipedia.org/wiki/Do_Not_Track) untranslated (without or without a translation in parenthesis) as it is a proper noun?